### PR TITLE
Conversions between base::JointLimits and KDL::jntArray and base::samples::Joints and KDL::JntArray added

### PR DESCRIPTION
**Pull request by rohitmenon86**

### CODE REVIEW

File: src/KDLConversions.cpp [hunk 1]
```diff
@@ -48,4 +48,78 @@ void WrenchToKDLWrench(const base::samples::Wrench& in, KDL::Wrench& out){
     BaseVector3dToKDLVector(in.torque, out.torque);
 }
 
+void baseJointLimits2KDLJntLimitArray(const base::JointLimits& jointLimits, KDL::JntArray& qMin, KDL::JntArray& qMax)
```
Comment (doudou) [inline src/KDLConversions.cpp]: When modifying a package, stick to the package's style. Here, it is 4 space indentation as you can see in the methods above.

File: src/KDLConversions.hpp [hunk 2]
```diff
@@ -18,14 +20,8 @@ void RigidBodyState2KDL(const base::samples::RigidBodyState& in, KDL::Frame& out
...
 void RigidBodyState2KDL(const base::samples::RigidBodyState& in, KDL::Frame& pose_out, KDL::Twist& twist_out);
 
-void baseJointLimits2KDLJntLimitArray(const std::string& startJoint, const std::string& endJoint,const base::JointLimits& jointLimits, KDL::JntArray& qMin, KDL::JntArray& qMax);
-void baseJointLimits2KDLJntLimitArray(const base::JointLimits& jointLimits, KDL::JntArray& qMin, KDL::JntArray& qMax);
-
-void baseJoints2KDLJntArray(const std::string& startJoint, const std::string& endJoint, const base::samples::Joints& jointSamples, KDL::JntArray& jntArray);
-void baseJoints2KDLJntArray(const base::samples::Joints& jointSamples, KDL::JntArray& jntArray);
-
-void KDLJntArray2baseJoints(const std::string& startJoint, const std::string& endJoint, const KDL::JntArray& jntArray, base::samples::Joints& jointSamples);
-void KDLJntArray2baseJoints(const KDL::JntArray& jntArray, base::samples::Joints& jointSamples);
```
Comment (jmachowinski) [inline src/KDLConversions.hpp]: These methods still exist, why did you remove the declaration ?
